src/scripts/qa.ts

```diff
@@ -20,7 +20,10 @@ import yargs from 'yargs/yargs';
 const parser = yargs(process.argv.slice(2)).options({
   maker: { string: true, demandOption: true },
   chain: { number: true, demandOption: true },
-  chainType: { string: true, default: 'evm' },
+  chain_type: { string: true, default: 'evm' },
```

Comment:
if we use chainType in neptune promote script make sure we update there as well

Reply:
Yeah thanks, planning to update that once this lands.